.github/profile/README.md

```diff
@@ -0,0 +1,23 @@
+## Welcome to our power grid full of...
+
+<img src="phlowers_logotxt.png" width="200" height="200" style="display: block; margin: 0 auto"/>
+
+
+🙋‍♀️ A short introduction - what is our organization all about?
+
+Phlowers is an open-source library suite for studying power lines, written in Python.
+
+The suite includes:
+- **mecaphlowers**: a package dedicated to mechanical and geometrical calculations for overhead power lines.
+- **thermohl**: a package focused on thermal analysis of power lines.
```

Comment:
```suggestion
- **thermohl**: a package focused on thermal analysis of overhead lines.
```

Reply:
It seems that you have change somewhere else for "overhead power lines", so we have to choose...